### DISCUSSION TITLE
Getting an Error on Lesson 11 Interactions: script failed: revert: No deployment artifacts were found for specified chain

### DISCUSSION THREAD

**Discussion by APexBlockchainDevelopments** [Cyfrin/foundry-full-course-cu#1319] [Q&A]
Hey everyone, was hoping for a bit of help on this here. I went ahead and ran the basic DeployBasicNft script. And seems like it's working properly. Contract is up on sepolia.etherscan and I can manually interact with it using cast send and cast call. I have a broadcoast folder with the json info in it. I can view the addresses and the chainid. 
However when I run my interactions script I get an issue of 
script failed: revert: No deployment artifacts were found for specified chain

Here is my interactions script 

```
import {DevOpsTools} from "../lib/foundry-devops/src/DevOpsTools.sol";

    function run() external {
        address mostRecentlyDeployedNFT = DevOpsTools.get_most_recent_deployment("BasicNft", block.chainid);
        mintNftOnContract(mostRecentlyDeployedNFT);
    }
```

I also slightly modified the DevOpsTools.sol file to log the file path. 

== Logs ==
  D:/Cyfrin/Lesson 11/broadcast\DeployBasicNft.s.sol
  D:/Cyfrin/Lesson 11/broadcast\DeployBasicNft.s.sol\11155111
  D:/Cyfrin/Lesson 11/broadcast\DeployBasicNft.s.sol\11155111\run-1705042180.json
  D:/Cyfrin/Lesson 11/broadcast\DeployBasicNft.s.sol\11155111\run-1705042193.json
  D:/Cyfrin/Lesson 11/broadcast\DeployBasicNft.s.sol\11155111\run-latest.json

It seems like it's looking in the right spot. My file hierarchy matches where it seems to be looking. 
Any help is appreciated, thanks!

**Comment by ghost** [comment]
I think the problem are the backlashes, it should be forward slashes. First check you're running the correct command to deploy, otherwise play with the conditions in the if statement in `DevOpsTool.sol` to see if you manage to make it work.

**Reply by APexBlockchainDevelopments** [reply]
Thanks man! That's a great idea, I'll take a look see if I can get it working.
